Service/CreateMarriageService.php

```diff
@@ -149,54 +160,90 @@ private function validateCeremonie(array $huwelijk)
         if (isset($huwelijk['ceremonie'])) {
             if (!$ceremonieProductObject = $this->entityManager->getRepository('App:ObjectEntity')->find($huwelijk['ceremonie'])) {
                 isset($this->io) && $this->io->error('huwelijk.ceremonie not found in the databse with given id');
+                $this->logger->error('huwelijk.ceremonie not found in the databse with given id');
 
                 return ['response' => ['message' => 'huwelijk.ceremonie not found in the databse with given id'], 'httpCode' => 400];
             }
 
             if (!in_array($ceremonieProductObject->getValue('upnLabel'), ['gratis trouwen', 'flits/balliehuwelijk', 'eenvoudig huwelijk', 'uitgebreid huwelijk'])) {
                 isset($this->io) && $this->io->error('huwelijk.ceremonie.upnLabel is not gratis trouwen, flits/balliehuwelijk, eenvoudig huwelijk, uitgebreid huwelijk');
+                $this->logger->error('huwelijk.ceremonie.upnLabel is not gratis trouwen, flits/balliehuwelijk, eenvoudig huwelijk, uitgebreid huwelijk');
 
                 return ['response' => ['message' => 'huwelijk.ceremonie.upnLabel is not gratis trouwen, flits/balliehuwelijk, eenvoudig huwelijk, uitgebreid huwelijk'], 'httpCode' => 400];
             }
 
             return true;
         } else {
             isset($this->io) && $this->io->error('huwelijk.ceremonie is not given');
+            $this->logger->error('huwelijk.ceremonie is not given');
 
             return ['response' => ['message' => 'huwelijk.ceremonie is not given'], 'httpCode' => 400];
         }
-
-        return true;
     }//end validateCeremonie()
 
     /**
      * This function creates a person object for the given user.
      */
-    private function createPerson(): ?ObjectEntity
+    private function createPerson(array $huwelijk, ?ObjectEntity $brpPerson = null): ?ObjectEntity
     {
         $personSchema = $this->getSchema('https://klantenBundle.commonground.nu/klant.klant.schema.json');
 
-        // @TODO BRP person has to be set to the klantObject
-        // @TODO get user/ person from jwt token and create a person object
+        if ($brpPerson) {
```

Comment:
if ($brpPerson !== null)
or
if ($brpPerson instanceof ObjectEntity === true)

(for codacy and) better readability of your code